lib/src/ruby/module.cc

```diff
@@ -4,6 +4,7 @@
 #include <internal/ruby/confine.hpp>
 #include <internal/ruby/simple_resolution.hpp>
 #include <facter/facts/collection.hpp>
+#include <facter/logging/logging.hpp>
```

Comment:
Nope. I tried that and everything blew up. This does not play nice with just having facter/logging included.